src/components/Button.js

```diff
@@ -1,11 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import styles from './Button.scss';
+import styles from '../components/Button.scss';
 
-const Button = ({ onClick, children, tabIndex, ...props }) =>
-(
+
+const buttonType = {
+  SQAURE_B: `${styles.square} ${styles.square_button}`,
+  ROUND_B: `${styles.round} ${styles.roundblue_button}`,
+  ROUND_BH: `${styles.round} ${styles.roundblue_button_header}`,
+  BORDER_B: `${styles.square} ${styles.blueborder_button}`,
+};
+
+const Button = ({ button, onClick, children, tabIndex, ...props }) => (
   <button
-    className={styles.Button}
+    className={`${styles.general_button} ${buttonType[button]}`}
```

Comment:
`general_button`: This can be done entirely in scss with [mixins](http://thesassway.com/intermediate/understanding-placeholder-selectors).